data/trie/leafNode_test.go

```diff
@@ -406,9 +441,10 @@ func TestLeafNode_insertInDirtyLnAtDifferentKey(t *testing.T) {
 
 func TestLeafNode_insertInNilNode(t *testing.T) {
 	t.Parallel()
+
 	db := mock.NewMemDbMock()
 	var ln *leafNode
-	node := newLeafNode([]byte{0, 2, 3}, []byte("dogs"))
+	node := newLeafNode([]byte("dog"), []byte("dogs"))
```

Comment:
why _dog_ ? it doesn't seem to match the original test values

Reply:
Same as above.